extensions/amp-sidebar/0.1/test/integration/test-amp-sidebar.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
```

Comment:
nit: 2018. But I am unsure how much it matters

Reply:
Will need to merge / rebase off of https://github.com/ampproject/amphtml/pull/12700 for that. =) I'll make this update when the rest of this pr is gtg.